torch_xla/csrc/aten_xla_type.cpp

```diff
@@ -2388,14 +2399,14 @@ at::Tensor& AtenXlaType::rsqrt_(at::Tensor& self) {
 
 at::Tensor AtenXlaType::rsub(const at::Tensor& self, const at::Tensor& other,
                              at::Scalar alpha) {
+  sub_check(self, other);
   return bridge::AtenFromXlaTensor(XLATensor::rsub(
       bridge::GetXlaTensor(self), bridge::GetXlaTensor(other), alpha));
 }
 
 at::Tensor AtenXlaType::rsub(const at::Tensor& self, at::Scalar other,
                              at::Scalar alpha) {
-  return bridge::AtenFromXlaTensor(
-      XLATensor::rsub(bridge::GetXlaTensor(self), other, alpha));
+  return rsub(self, c10::scalar_to_tensor(other, self.device()), alpha);
```

Comment:
Do not route this to the previous rsub().
We have special logic to convert scalars to tensors.
Same thing below.